### PR TITLE
Adding retags to mention notifications

### LINKED ISSUE #413: mention notifs should show tags
it would be sick if this showed tags as well as content:
image
(also on the dashboard as well)

### PR DESCRIPTION
**Pull request by blackjackkent**
Fixes #413

### CODE REVIEW

File: Extensions/retags.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
 XKit.extensions.retags = {
 	running: false,
 	api_key: '3DFxEZm0tGISOmdvWe9Fl1QsQMo1LFqEatnc8GQ68wgF1YTZ4w',
-	selectors: '.reblog,.is_reblog,.notification_reblog,.is_reply,.is_answer',
+	selectors: '.reblog,.is_reblog,.notification_reblog,.is_reply,.is_answer,.is_user_mention,.notification_user_mention',
```
Comment (nightpool) [inline Extensions/retags.js]: lol nice one line fix.

File: Extensions/retags.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 //* TITLE       Retags **//
 //* DEVELOPER   alexhong **//
-//* VERSION     0.6.9 **//
+//* VERSION     0.7.0 **//
```
Comment (nightpool) [inline Extensions/retags.js]: why not 6.10?

Comment (nightpool) [inline Extensions/retags.js]: actually changed my mind. just push this to 1.0.0. we should have done that before anyway

Comment (Wolvan) [inline Extensions/retags.js]: Should we change developer as well while we're at it?

**Comment by Wolvan** [comment]
Also, we might wanna check travis

**Comment by Wolvan** [comment]
Worked perfectly for me, lgtm
@homu r+

**Pull request merged**